### DISCUSSION TITLE
My application is approved in the web,but the email I received was rejected.

### DISCUSSION THREAD

**Discussion by Kouke233** [community/community#145095] [GitHub Education] [answered]
### Hi everyone,

My application is approved in the web,but the email I received was rejected.Which one is right?Need I apply for it again?
![829d4a2a4454bc0e81e48d893a2dfd2a](https://github.com/user-attachments/assets/ae4053ac-9d31-4bb9-a7ea-c7fa2bfb7a4b)
![c89bc6307c88bf77f83f2e3433da6ecf](https://github.com/user-attachments/assets/6ac830cd-20cf-464b-869a-dcdf4850f74a)

**Comment by Kouke233** [comment]
And I used both OBS visual camera (because the laptop camera only 720P.It's hard to take my file clearly) and my laptop camera,but the email told me like this.

**Comment by coolguy-kr** [comment, marked as answer]
Disregard that email.

**Reply by Kouke233** [reply]
Thanks.I'll continue to wait for my educationbenefits.

**Reply by coolguy-kr** [reply]
@Kouke233 
Yes, the reason why you got that mail seems the reviewer just got a fat finger and fixed it.

**Reply by Kouke233** [reply]
I see.Thank you very much.
